collection-scripts/gather_logs

```diff
@@ -41,6 +41,11 @@ for ns in ${namespaces[@]}; do
     migmigration=$(oc get restore $restore -o jsonpath='{.metadata.labels.migration\.openshift\.io/migmigration-name}' --namespace $ns)
     object_collection_path="/must-gather/migplan-${migplan}/migmigration-${migmigration}/restore-${restore}/pvr-${pvr}"
     mkdir -p ${object_collection_path}
+    oc get migplan ${migplan} -o yaml --namespace ${ns} &> "/must-gather/migplan-${migplan}/${migplan}.yml"
```

Comment:
I don't think adding these under the PVR loop is the right thing, since not every migration will have PVRs or PVBs.

Reply:
Good catch! I updated the code